doc/operation-and-maintenance/Reloading-configuration-on-a-running-system.md

```diff
@@ -26,10 +26,11 @@ Useful for debugging.
 ### Non-reloadable options
 Some options require restarting the server in order to be reloaded.
 The following options' changes will be ignored when using `mongooseimctl` tool:
+
 * domain_certfile
-* s2s_*
+* s2s_\*
 * all_metrics_are_global
-* rdbms_*
+* rdbms_\*
 
 
 ### Node-specific options
```

Comment:
This section is not supported in TOML. I think we should note that:
- it can be used only in the 'cfg' format
- it is deprecated

I think that MongooseIM should know which options can have different options per node (they are in `local_config`) and allow for different values here.
Alternatively, such allowance could be opt-in and require an option to `reload cluster`.
This would be a new task of course.

Reply:
> MongooseIM should know which options can have different options per node

It's installation specific. For some clients, DB connection options could be cluster-specific.
But some clients want each server to use a different DB username.

In the perfect world, there is no node-specific options, all nodes are the same. Except some cases when clients want to customize some weird option per host.

Reply:
> > MongooseIM should know which options can have different options per node
> 
> It's installation specific. For some clients, DB connection options could be cluster-specific.
To clarify: what I meant is that the options in `config` are NOT node-specific and this does not change between installations. Btw, you probably meant 'node-specific', not 'cluster-specific'.

The ones in 'local_config' - I would just allow them to differ by default for simplicity, maybe with an on/off switch for all of them if needed.

> But some clients want each server to use a different DB username.
I guess you mean 'each node' - the logic above would be enough for them.

> In the perfect world, there is no node-specific options, all nodes are the same. Except some cases when clients want to customize some weird option per host.
> It's also _extremely_ edge case and _not recommened_ case.
Looks like my suggestion is valid then.

Reply:
I'm not sure if I follow the discussion, so I deleted most of the problematic file. Not sure about the `Non-reloadable options` and how these relate to TOML (if they do at all) though...